app/Controllers/LoanApprovalController.php

```
@@ -192,7 +192,67 @@ public function approveLoan(Request $request, Response $response, mysqli $db): R
             $existingCopiaId = $loan['copia_id'] ? (int) $loan['copia_id'] : null;
             $dataPrestito = $loan['data_prestito'];
             $dataScadenza = $loan['data_scadenza'];
-            $today = date('Y-m-d');
+            // "Oggi" calcolato nel timezone configurato dell'applicazione, non in quello
+            // di default di PHP (spesso UTC in produzione): altrimenti, vicino alla
+            // mezzanotte, un prestito datato oggi (lato browser/Rome) verrebbe visto come
+            // futuro e finirebbe 'prenotato' invece di 'da_ritirare' (P1 timezone).
+            $appTz = \App\Support\ConfigStore::get('app.timezone', 'Europe/Rome');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\ConfigStore' in method 'approveLoan'. (undefined)

(StaticAccess)